### PR TITLE
Consistent indentation in status examples

### PR DESCRIPTION
**Pull request by imhardikj**
Partially Addresses #1824:

> - [ ] `status` examples in all docs (including consistent indentation — may keep tabs)

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
@imhardikj is this ready for review? I see it's still in draft status.

File: content/docs/command-reference/status.md [hunk 1]
```diff
@@ -156,24 +156,24 @@ that.
...
+                new:                bar
         always changed
 ```
 
-This shows that for stage `bar.dvc`, the dependency `foo` and the
-<abbr>output</abbr> `bar` have changed. Likewise for `foo.dvc`, the dependency
-`foo` has changed, but no output has changed.
+This shows that for stage `bar`, the dependency `bar` and the
+<abbr>output</abbr> `foo` have changed. Likewise for `foo.dvc`, the output `foo`
+has changed, but no dependency has changed.
```
Comment (imhardikj) [inline content/docs/command-reference/status.md:176]: @jorgeorpinel didn't make it ready for review as many PRs were open.
Currently I have changed indentation in only 3 files. I have kept a tab (8 spaces) for indentation. If this is correct I'll do it for all docs.
Also this example in `status.md` looked outdated so changed this too.

Comment (jorgeorpinel) [reply content/docs/command-reference/status.md:176]: Yes you're already working on 2 PRs so let's focus on those please. Or do you have any extra/idle time to do this too?

File: content/docs/command-reference/status.md [hunk 2]
```diff
@@ -156,24 +156,24 @@ that.
 
 ```dvc
 $ dvc status
-bar.dvc:
-        changed deps:
-                modified:      bar
+bar:
+	changed deps:
+        	modified:           bar
```
Comment (jorgeorpinel) [inline content/docs/command-reference/status.md:161]: As for the changes. If this is what DVC outputs, then it is correct. We'll trust you on this one to actually run each commit.

**Comment by jorgeorpinel** [comment]
But again, since there's 2 active PRs I'll close this for now. Feel free to keep the branch of course and keep working on it locally for when there's a chance to reopen this. Thanks

**Pull request closed without merging**